.github/workflows/firesim-run-tests.yml

```diff
@@ -414,3 +421,10 @@ jobs:
             git checkout .
             exit 1
           fi
+
+      # Run clang-tidy on the entire codebase. Error will be logged.
+      - name: Run clang-tidy
+        run: |
+          export FIRESIM_ENV_SOURCED=1
+          export FIRESIM_STANDALONE=1
+          make -C sim clang-tidy
```

Comment:
Does `clang-tidy -fix` exit non-zero if it fixes errors or will it exit zero with changes made to files?  In other words, in the CI, do we also need to add a check for modified files after running clang-tidy?